app/lib/rack_exceptions_app.rb

```diff
@@ -0,0 +1,18 @@
+class RackExceptionsApp
+  def self.call(env)
+    if env['REQUEST_PATH'] =~ /^\/[^\/]*api\/.*$/
+      [500, { 'Content-Type' => 'text/plain' }, ['Internal Server Error']]
```

Comment:
```suggestion
      [500, { 'Content-Type' => 'application/json' }, ['Internal Server Error']]
```

Reply:
'Internal Server Error' is not valid JSON. We could work around this by returning `'"Internal Server Error"'`. What do you think?

Reply:
See https://stackoverflow.com/a/25415850

Reply:
discussed in person and agreed to go with `application/json` and the valid string